### ISSUE TITLE
command sortbysize does not accept the sizein option anymore

### ISSUE THREAD

**Issue by frederic-mahe** [torognes/vsearch#385]
With the latest vsearch version, `--sortbysize` does not accept the `--sizein` option anymore:
```sh
printf ">s;size=1\nA\n" | \
     vsearch --sortbysize - --sizein --sizeout --output -
```
```
Fatal error: Invalid options to command sortbysize
Invalid option(s): --sizein
The valid options for the sortbysize command are: --bzip2_decompress --fasta_width --fastq_ascii --fastq_qmax --fastq_qmin --gzip_decompress --log --maxseqlength --maxsize --minseqlength --minsize --no_progress --notrunclabels --output --quiet --relabel --relabel_keep --relabel_md5 --relabel_sha1 --sizeout --threads --topn --xee --xsize
```
If I am not mistaken, `--sortbysize` should accept `--sizein`. I've tried to use git's blame command, but I was not able to pinpoint the commit that changed that behavior.

**Comment by torognes** [comment]
Fixed in commit 11a074e8863c87b514f9783708fdedcc868d2841.

**Comment by frederic-mahe** [comment]
The toy-example now passes. Issue fixed!

**Comment by torognes** [comment]
Fixed in version [2.14.0](https://github.com/torognes/vsearch/releases/tag/v2.14.0) just released.
